packages/contracts/src/dollar/mocks/MockBondingV1.sol

```diff
@@ -1,4 +1,5 @@
 // SPDX-License-Identifier: MIT
+// cspell: disable // disabled because all of bonding v1 is deprecated
```

Comment:
This file might need to be removed?

Reply:
This was for testing the migration from V1 to V2 when I was doing the investigation.